wrangler-core/src/main/java/io/cdap/directives/aggregates/SetTransientVariable.java

```diff
@@ -82,18 +82,7 @@ public void destroy() {
   public List<Row> execute(List<Row> rows, ExecutorContext context) throws DirectiveExecutionException {
     for (Row row : rows) {
       // Move the fields from the row into the context.
-      ELContext ctx = new ELContext(context);
```

Comment:
If this is moved to EL class, does it mean custom directives will need to remove this if they are using it and recompile?

Reply:
These are advanced, I haven't see any one us these type. EL is not exposed outside.